docs/workflows/message_granule_writes.md

```diff
@@ -35,7 +35,7 @@ For each granule to be written, the following constraints apply:
 
   Granule write will not be allowed if `granuleId` already exists in the database for another collection, granules in this state will be rejected to write and wind up in the [Dead Letter Archive](https://nasa.github.io/cumulus/docs/features/dead_letter_archive/)
 
-- Message granule must match the [API Granule schema](https://github.com/nasa/cumulus/blob/master/packages/api/lib/schemas.js).
+- Message granule must match the [API Granule schema](https://github.com/nasa/cumulus/blob/master/packages/api/lib/schemas.js), except `producerGranuleId` is optional. If `producerGranuleId` is set to `granuleId` if it's absent in the message granule.
```

Comment:
This looks good, but we need to add the new field to the ## Message Granule Write Behavior table below.